crates/uv-resolver/src/resolver/mod.rs

```diff
@@ -3120,8 +3117,5 @@ fn satisfies_requires_python(
 /// Returns true if and only if the given requirement's marker expression has a
 /// possible true value given the `markers` expression given.
 fn possible_to_satisfy_markers(markers: &MarkerTree, requirement: &Requirement) -> bool {
```

Comment:
Unrelated to this changed, but also noticed that the invariant here is reversed. This should be fixed as part of https://github.com/astral-sh/uv/issues/5562.